lib/services/local/users.go

```diff
@@ -907,6 +907,16 @@ func (s *IdentityService) GetWebauthnSessionData(ctx context.Context, user, sess
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
+
+	if s.Clock().Now().UTC().After(item.Expires) {
+		// Webauthn session already expired. Some backends do not clean up expired
+		// items in a timely manner, force delete.
+		if err := s.Delete(ctx, item.Key); err != nil {
```

Comment:
I guess it would be semi likely to get a NotFound with any concurrency
```suggestion
		if err := s.Delete(ctx, item.Key); err != nil && !trace.IsNotFound(err) {
```